src/Raven.Server/Documents/Handlers/RevisionsHandler.cs

```diff
@@ -383,22 +383,19 @@ private async Task GetRevisions(DocumentsOperationContext context, bool metadata
 
             HttpContext.Response.Headers["ETag"] = "\"" + actualChangeVector + "\"";
 
-            long loadedRevisionsCount;
-            long totalDocumentsSizeInBytes;
             await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
             {
                 writer.WriteStartObject();
                 writer.WritePropertyName("Results");
-                (loadedRevisionsCount, totalDocumentsSizeInBytes) = await writer.WriteDocumentsAsync(context, revisions, metadataOnly, token);
+                var (loadedRevisionsCount, totalDocumentsSizeInBytes) = await writer.WriteDocumentsAsync(context, revisions, metadataOnly, token);
+                AddPagingPerformanceHint(PagingOperationType.Revisions, nameof(GetRevisions), HttpContext.Request.QueryString.Value, loadedRevisionsCount, pageSize, sw.ElapsedMilliseconds, totalDocumentsSizeInBytes);
```

Comment:
I would prefer to leave the EP as, so the duration will account also the network here.
Instead let's retry to assert this in the test

Reply:
Done